apps/src/templates/projects/ProjectCard.jsx

```diff
@@ -35,9 +205,217 @@ export default class ProjectCard extends React.Component {
       isPublicGallery && isDetailView && projectData.publishedAt;
     const noTimeOnCardStyle = shouldShowPublicDetails ? {} : styles.noTime;
 
+    const {
+      showReportAbuse,
+      isAnyCheckboxSelected,
+      checkboxes,
+      showSubmittedHeader,
+      submitButtonEnabled,
+      reportButtonHovered,
+      reportButtonPressed,
+      showSubmitConfirmation,
+    } = this.state;
+
+    const captchaSiteKey = getScriptData('projects').recaptchaSiteKey;
+
+    const script = document.createElement('script');
+    script.src = 'https://www.google.com/recaptcha/api.js';
+    script.id = 'captcha';
+    script.async = true;
+    script.defer = true;
+    window.onCaptchaSubmit = token => this.onCaptchaVerification(token);
+    window.onCaptchaExpired = () => this.onCaptchaExpiration();
+    script.onload = () => this.setState({loadedCaptcha: true});
+    document.body.appendChild(script);
+
     return (
       <div className="project_card">
+        {showReportAbuse ? (
+          <AccessibleDialog
+            onClose={this.cancel}
+            className={style.reportAbusePopUp}
+          >
+            {showSubmitConfirmation ? (
+              <div
+                style={{
+                  display: 'flex',
+                  flexDirection: 'column',
+                  alignItems: 'center',
+                  justifyContent: 'center',
+                }}
+              >
+                <h3 style={{margin: 10}}>Thank you!</h3>
+                <p style={{margin: 10}}>
+                  Thanks for helping us to keep Code.org safe!
+                </p>
+                <Button
+                  onClick={this.cancel}
+                  text={'Continue'}
+                  color={Button.ButtonColor.brandSecondaryDefault}
+                  style={{margin: 10}}
+                />
+              </div>
+            ) : (
+              <div>
+                <div
+                  style={{
+                    display: 'flex',
+                    justifyContent: 'space-between',
+                    alignItems: 'center',
+                  }}
+                >
+                  <h3 style={{margin: 0}}>Report Abuse</h3>
+                  <button
+                    type="reset"
+                    onClick={this.cancel}
+                    style={{
+                      ...styles.transparentButton,
+                      width: 24,
+                      height: 24,
+                      fontSize: 16,
+                      padding: 5,
+                    }}
+                  >
+                    <FontAwesome icon="x" style={{color: '#D4D5D7'}} />
+                  </button>
+                </div>
+                <hr style={styles.popUpLines} />
+                <p style={styles.popUpBody}>
+                  Why are you reporting this content?
+                </p>
+                <div>
+                  {checkboxes.map(checkbox => (
```

Comment:
For checkboxes please use existing designSystem `Checkbox` component (see apps/src/componentLibrary/checkbox/Checkbox.tsx)